dominic_monyoncho/queue/main.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>//input/output
```

Comment:
After second thought, the commit message is horrible and must have been a product of uncalled for haste.Any feedback improved is most welcomed